packages/Button/src/Button.styles.js

```diff
@@ -28,11 +28,17 @@ const commonStyles = `
     outline: none;
   }
 
+  [data-whatinput="mouse"] &:focus {
+    box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.1);
+    border-color: ${tokens.border.color};
+  }
```

Comment:
Need to be a little more selective about this... it actually shouldn't be applied to `minor` or `link` buttons.  And the `box-shadow` probably shouldn't be applied to the `flat` buttons. 

![Screen Shot 2020-01-03 at 10 32 02 AM](https://user-images.githubusercontent.com/14944896/71741670-5ba53a00-2e14-11ea-9c2c-42fce0df5cb6.png)
![Screen Shot 2020-01-03 at 10 31 56 AM](https://user-images.githubusercontent.com/14944896/71741671-5ba53a00-2e14-11ea-9edc-23d824c6bf78.png)